test/utils_test.py

```diff
@@ -0,0 +1,92 @@
+from shopify.utils import *
+from test.test_helper import TestCase
+from datetime import datetime, timedelta
+
+import jwt
+import sys
+
+if sys.version_info[0] < 3:  # Backwards compatibility for python < v3.0.0
+    import time
+
+    def timestamp(date):
+        return time.mktime(date.timetuple())
+
+
+else:
+
+    def timestamp(date):
+        return date.timestamp()
+
+
+class UtilsTest(TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.secret = "API Secret"
+        self.api_key = "API key"
+
+    @classmethod
+    def setUp(self):
+        current_time = datetime.now()
+        self.payload = {
+            "iss": "https://test-shop.myshopify.com/admin",
+            "dest": "https://test-shop.myshopify.com",
+            "aud": self.api_key,
+            "sub": "1",
+            "exp": timestamp((current_time + timedelta(0, 60))),
+            "nbf": timestamp(current_time),
+            "iat": timestamp(current_time),
+            "jti": "4321",
+            "sid": "abc123",
+        }
+
+    def test_SessionTokenUtility_getDecodedSessionToken_raises_if_auth_header_is_not_Bearer(self):
```

Comment:
can you move all of the functions with the `test_SessionTokenUtility_getDecodedSessionToken` prefix into its own class? So it would be something like
```python3
class TestSessionTokenUtilityGetDecodedSessionToken(TestCase):
    def test_raises_if_auth_header_is_not_Bearer(self):
        pass
```

or you can just change
```python3
class UtilsTest(TestCase):
```
on line 21 to the new class name

Reply:
For sure! I'd also love to hear your thoughts on how we can organize this file + class for extensibility. Ideally, I foresee adding more utility methods + classes in the future. 

I'm unsure whether they should belong in this file, or in other files under `/utilities` or something. Do you have any thoughts on this?

Reply:
Now that you mention it, our tests are not sorted at all 😆 ideally, we would have our tests mirror the directory structure of the other files, especially since we're introducing `utils_test.py` in `shopify/`. It would look like this
`shopify/resources/`=> `test/shopify/resources/`
`shopify/utils_test.py` => `test/shopify/`
I would want a second opinion from @mllemango but I think you can leave the test in `test/` and later we can open a PR to move all the tests.

In terms of organizing the classes, I like creating a new class per function that we test. So the classes would be like
```python3
class TestNameOfFunction(TestCase):
    def test_success(self):
        pass
```